.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ThirdParty/units"]
+	path = ThirdParty/units
+	url = https://github.com/mcneish1/units
```

Comment:
is this going to work since it is a private repo yet?

Reply:
Yep, when you update your submodules, it should prompt you for a github.com password

Reply:
how will that work with Travis?

Reply:
might be best to just stick the code in a thirdParty folder and build it with the utilities library until the library gets a little more settled.

Reply:
There's no issue once units is allowed to be public. Travis is probably cloning recursively already, and if not that's an easy thing to fix.